.github/workflows/run-tests.yml

```
@@ -3,14 +3,40 @@ run-name: "Run Tests: ${{ github.ref_type }} ${{ github.ref_name }} by ${{ githu
 
 on:
   workflow_dispatch:
-  # pull_request:
-  #   branches: [ main ]
-  # push:
-  #   branches: [ main ]
-  # schedule:
-  #   - cron: "21 11 * * 0"
+  pull_request:
+    branches: [ main ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)